src/sections/Community/Handbook/repository.js

```diff
@@ -30,6 +32,7 @@ const Repository = () => {
     <HandbookWrapper>
       <div className="page-header-section">
         <h1>Repository Overview</h1>
+        <p>A brief introduction to all of the Github repositories of Layer5 organization is offered below. This overview serves as a resource to newcomers seeking a perspective of the collective community efforts (from a code-centric perspective). See <a href="https://docs.google.com/document/u/0/d/1FzX-C_xy9hZ3Eu9dcCE0unhFV5LRUrb5YLn_MGYuG6Y/edit">Layer5 GitHub Process</a> for more related information.</p>
```

Comment:
![SharedScreenshot](https://user-images.githubusercontent.com/85789734/147653608-a43ed677-d93f-4717-a8f0-96c0319b3728.jpg)
This paragraph should go inside `<div className="content"></div>`

Reply:
Thank you, @debo19!